### PR TITLE
Tiw128

### PR DESCRIPTION
**Pull request by AshenCat**

### CODE REVIEW

File: src/contexts/ProductsContext.js [hunk 1]
```diff
@@ -115,6 +115,7 @@ export const ProductsProvider = (props) => {
 
   useEffect(() => {
     getProducts();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (jacildoaldrin) [inline src/contexts/ProductsContext.js:118]: Hmm, is it showing a warning again?

**Review by jacildoaldrin — APPROVED** [review]

File: src/contexts/ProductsContext.js (hunk 1 above)
Comment (AshenCat) [reply src/contexts/ProductsContext.js:118]: yeah

Comment (lilianyangc) [reply src/contexts/ProductsContext.js:118]: olah

Comment (AshenCat) [reply src/contexts/ProductsContext.js:118]: (☞ﾟヮﾟ)☞

**Review by lilianyangc — APPROVED** [review]

**Review by victafrancis — APPROVED** [review]
🌵 🌞

**Pull request merged**